src/pages/Mint.js

```diff
@@ -34,7 +39,7 @@ export default function Mint({storiesState}) {
   }
 
   function Mint() {
-    console.log(story)
+    mint()
   }
```

Comment:
not sure what this Mint() is doing

Reply:
Nothing as of now, will implement the mint function by the next meeting,